earthfile2llb/converter.go

```diff
@@ -1672,7 +1678,7 @@ func (c *Converter) internalRun(ctx context.Context, opts ConvertRunOpts) (pllb.
 	if !opts.Locally {
 		// Debugger.
 		secretOpts := []llb.SecretOption{
-			llb.SecretID(common.DebuggerSettingsSecretsKey),
+			llb.SecretID(c.secretID(common.DebuggerSettingsSecretsKey)),
```

Comment:
Something feels a bit odd with wrapping this var to potentially include an org and project -- only to then ignore it.

Maybe we should have a `c.secretIDForceLocal()`, or even `c.secretIDHack()`.

This is a nit-pick comment, and eventually the debugger should be reworked to avoid using ports and then this hack will go away. Ultimately if the tests are passing, I'm ok with this change :)